<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>姓名：</string>
     </property>
     <property name="buddy">
      <cstring>nameLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="nameLineEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="sexLabel">
     <property name="text">
      <string>性别：</string>
     </property>
     <property name="buddy">
      <cstring>sexComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="sexComboBox">
     <item>
      <property name="text">
       <string>男</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>女</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="ageLabel">
     <property name="text">
      <string>年龄：</string>
     </property>
     <property name="buddy">
      <cstring>ageSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="ageSpinBox"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="phoneLabel">
     <property name="text">
      <string>电话：</string>
     </property>
     <property name="buddy">
      <cstring>phoneLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="phoneLineEdit"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="adressLabel">
     <property name="text">
      <string>地址：</string>
     </property>
     <property name="buddy">
      <cstring>phoneLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="adressLineEdit"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="idCardLabel">
     <property name="text">
      <string>身份证号：</string>
     </property>
     <property name="buddy">
      <cstring>phoneLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="idCardLineEdit"/>
   </item>
   <item row="6" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="moreBtn">
       <property name="text">
        <string>更多</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="registerBtn">
       <property name="text">
        <string>注册</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>label</zorder>
  <zorder>lineEdit</zorder>
  <zorder>lineEdit_2</zorder>
  <zorder>label_2</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>nameLabel</zorder>
  <zorder>nameLineEdit</zorder>
  <zorder>sexLabel</zorder>
  <zorder>sexComboBox</zorder>
  <zorder>ageLabel</zorder>
  <zorder>ageSpinBox</zorder>
  <zorder>phoneLabel</zorder>
  <zorder>phoneLineEdit</zorder>
  <zorder>moreBtn</zorder>
  <zorder>registerBtn</zorder>
  <zorder>moreBtn</zorder>
  <zorder>adressLineEdit</zorder>
  <zorder>adressLabel</zorder>
  <zorder>idCardLabel</zorder>
  <zorder>idCardLineEdit</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
